llvm/lib/Transforms/Vectorize/VPlanUtils.h

```diff
@@ -240,6 +242,9 @@ class VPBlockUtils {
     VPBlockUtils::connectBlocks(From, BlockPtr, -1, SuccIdx);
     VPBlockUtils::connectBlocks(BlockPtr, To, PredIx, -1);
   }
+
+  /// Returns true if \p VPB is a loop header.
```

Comment:
```suggestion
  /// Returns true if \p VPBB is a loop header, based on regions or \p VPDT in their absence.
```

Reply:
Adjusted the name to `VPB` (arg is `VPBlockBase`), and update the erst, thanks